src/views/Player.vue

```diff
@@ -61,7 +61,8 @@ export default {
         'failsafeUrl': ''
       },
       enterFullscreen: [],
-      exitFullscreen: []
+      exitFullscreen: [],
+      sessionId: 1
```

Comment:
`sessionId` is always one? Or is there somewhere you are incrementing if file exists already on backend (didn't see in that PR)?

Reply:
It is getting updated. Tagged you in a comment pointing towards the code change.